src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -33,7 +34,7 @@ export const getClusterWizardFirstStep = (
     case 'pending-for-input':
     case 'adding-hosts':
     case 'insufficient':
-      return 'host-discovery';
+      return 'operators';
```

Comment:
I'm not very familiar with these operators, but just to confirm this is what we want.

If the cluster is insufficient, we will move to Operators and not to host discovery?
And also, I can see later that in Host Discovery we don't wait for `sufficient-masters-count` anymore. So users can access networking even though not all hosts were discovered?

Reply:
If I remember correctly, that's what we agreed. 
 @jkilzi  @nirfarkas  What do you think? To what step should we go if insufficient? 
And the sufficient-masters-count it's still a validation in host discovery.